server/app/views/v1/grid_services/_grid_service.json.jbuilder

```diff
@@ -21,7 +21,7 @@ json.secrets grid_service.secrets.as_json(only: [:secret, :name, :type])
 json.memory grid_service.memory
 json.memory_swap grid_service.memory_swap
 json.cpu_shares grid_service.cpu_shares
-json.volumes grid_service.volumes
+json.volumes grid_service.service_volumes.map {|sv| sv.to_s}
```

Comment:
yes, service itself does not anymore have the "raw" volume definitions thus the `ServiceVolume`s are stringified to have the API backwards compatible. I'll add a test case to verify it :)